include/linux/osd.h

```diff
@@ -95,14 +97,6 @@ static inline int ofi_hugepage_enabled(void)
 
 size_t ofi_ifaddr_get_speed(struct ifaddrs *ifa);
 
-#ifndef __NR_process_vm_readv
-# define __NR_process_vm_readv 310
-#endif
-
-#ifndef __NR_process_vm_writev
-# define __NR_process_vm_writev 311
-#endif
-
```

Comment:
Why remove this? The use case for this was to build on systems that might not have CMA but for use in systems that do.

Reply:
@rwespetal - please see commit fbaf6ceff052713f7ca9eb7d9ce653bf454c6fbb for why this was added.

Reply:
I don't think we should have hardcoded syscall numbers here. Can we change the functions to return ENOSYS when these symbols are not defined instead?

Reply:
The problem is that libfabric is being built on older systems, where these are not defined, but the binary is being used on systems where CMA is available.

Reply:
I see, thanks for the clarification. I updated the PR.